plugins/ua_config_standard.c

```diff
@@ -9,6 +9,7 @@
 #include "ua_securitypolicy_none.h"
 #include "ua_types.h"
 #include "ua_types_generated_handling.h"
+#include "ua_client_highlevel.h"
```

Comment:
I'd create a new PR based on master for the separate configs and also integrate it in this PR.

IMHO this PR can then be merged, but let's wait for @jpfr